mesheryctl/internal/cli/root/system/start.go

```diff
@@ -341,14 +341,24 @@ func start() error {
 		// downloaded required files successfully now apply the manifest files
 		log.Info("Starting Meshery...")
 
-		// apply the adapters mentioned in the config.yaml file to the Kubernetes cluster
-		err = utils.ApplyManifestFiles(manifests, RequestedAdapters, kubeClient, false, false)
+		spinner := utils.CreateDefaultSpinner("Deploying Meshery on Kubernetes.", "...Meshery deployed on Kubernetes.")
+		spinner.Start()
 
-		if err != nil {
-			return err
+		deadline := time.Now().Add(20 * time.Second)
```

Comment:
If the deployment happens faster than 20 seconds, the user will have to wait anyway, correct? Why not have a lower deadline or do away with the deadline altogether?

Reply:
@nagaharshita, What do you think about this? How do we optimize for the least amount of unnecessary wait time?

Reply:
How about we remove the deadline so that the spinner runs for the exact time till the Meshery is deployed?

Reply:
Yes, let the spinner run until Meshery is deployed. It should not be an issue, if any error occurs inbetween, it will stop the execution and show the error.

Reply:
@leecalcote It is actually the other way around. The maximum time we wait is 20 seconds. If we deploy faster than that, then the spinner stops. That is the way the deadline works (also how it is implemented in `system stop`), will have to check this PR out.

Reply:
Oh, I see. On a related note, we are allowing the user’s browser to open before Meshery Server is serving its UI. The check for this doesn’t seem to be working a new longer on either Docker or Kubernetes deployments.

Reply:
@metonymic-smokey ^^